package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "meeting-calendar",
+  "version": "1.0.0",
+  "description": "A meeting calendar for WordPress.org",
+  "main": "index.js",
+  "scripts": {
+    "build": "wp-scripts build",
+    "check-engines": "wp-scripts check-engines",
+    "check-licenses": "wp-scripts check-licenses",
+    "format:js": "wp-scripts format-js",
+    "lint:css": "wp-scripts lint-style",
+    "lint:js": "wp-scripts lint-js",
+    "lint:md:docs": "wp-scripts lint-md-docs",
+    "lint:md:js": "wp-scripts lint-md-js",
+    "lint:pkg-json": "wp-scripts lint-pkg-json",
+    "packages-update": "wp-scripts packages-update",
+    "start": "wp-scripts start",
+    "test:e2e": "wp-scripts test-e2e",
+    "test:unit": "wp-scripts test-unit-js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Automattic/meeting-calendar.git"
+  },
+  "author": "",
+  "license": "ISC",
```

```suggestion
  "license": "GPL-2.0-or-later",
```

I might also suggest `"private": true` so this can't be accidentally published to npm